<launch>
  <node name="orb_test_node" pkg="orb_test" type="orb_test_node" output="screen">
    <rosparam file="$(find orb_test)/config/common.yaml"/>
    <rosparam file="$(find orb_test)/config/san_rafael.yaml"/>
  </node>

  <node pkg="tf" type="static_transform_publisher" name="map_vio_tf" respawn="true"
        args="0 0 0 0 0 0 1 world vio_world 10 "/>
</launch>
